persistence/context.go

```diff
@@ -12,7 +12,7 @@ func (p PostgresContext) NewSavePoint(bytes []byte) error {
 
 func (p PostgresContext) RollbackToSavePoint(bytes []byte) error {
 	log.Println("TODO: RollbackToSavePoint not fully implemented")
-	return p.DB.Tx.Rollback(context.TODO())
+	return p.tx.Rollback(context.TODO())
```

Comment:
Choose/consolidate `tx` or `txn`, not both - to name this field

And use `p.GetTxn()` instead of accessing directly

Reply:
Done. I removed the error so let me know if that's kosher in your opinion or not.